bulk_insert.go

```diff
@@ -76,10 +76,16 @@ func insertObjSet(db *gorm.DB, objects []interface{}, excludeColumns ...string)
 		mainScope.SQLVars = append(mainScope.SQLVars, scope.SQLVars...)
 	}
 
-	mainScope.Raw(fmt.Sprintf("INSERT INTO %s (%s) VALUES %s",
+	insertOption := ""
+	if val, ok := db.Get("gorm:insert_option"); ok {
+		insertOption = val.(string)
+	}
+
```

Comment:
This `val` is not always string since any type of value set in `db.Set` is allowed. If this value is type of int, panic will be triggered.

To deal with this, I think it's better to detect whether the type assertion succeeded and then assign the value.

```suggestion
	insertOption := ""
	if val, ok := db.Get("gorm:insert_option"); ok {
		strVal, ok := val.(string)
		if ok {
			insertOption = strVal
		}
	}
```

Reply:
Sounds good. How about the case for `!ok`? Should we return an error to inform the caller that insert_option should be a string?

Reply:
This handling will be determined by whether it is regarded as the specification of this library or close to the original gorm. Either way, I think there isn't any absolutely right answer.f

In conclusion, I prefer to return an error if a non-string value is included. Because any value other than string is ineligible for this interface.

Of course, if you have other ideas, I'll consider them. I'm glad if you can think and modify this.